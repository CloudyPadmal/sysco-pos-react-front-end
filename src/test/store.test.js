import React from "react";
import { CookiesProvider } from "react-cookie";
import { Provider } from "react-redux";
import { mount } from "./enzyme";
import store from "../redux.store";

describe("Redux store", () => {
    it("imports redux store", () => {
        const wrapper = mount(
            <CookiesProvider>
                <Provider store={store} />
            </CookiesProvider>
        );
        expect(wrapper).toMatchSnapshot();
    });
});