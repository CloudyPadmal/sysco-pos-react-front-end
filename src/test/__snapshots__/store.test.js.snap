// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redux store imports redux store 1`] = `
<CookiesProvider>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  />
</CookiesProvider>
`;
